content/posts/2025-08-27-cli_ui_refactor/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: :theme/post
+title: "Network Observability On Demand UI Refactor"
+description: Enhanced command-line interface in NetObserv 1.10
+tags: CLI,Monitoring,Troubleshooting
+authors: [jpinsonneau]
+---
+
+The NetObserv 1.10 release introduces a sleek, interactive command-line interface powered by the excellent [tview library](https://github.com/rivo/tview) and one of its extension [tvxwidgets](https://github.com/navidys/tvxwidgets).
+
+These open-source Go packages bring a rich set of UI components such as buttons, dropdowns, popups and tables, all seamlessly integrated into your terminal.
+
+![tview](tview.gif)<br>
+Figure 1: Tview examples
+
+# Key Capabilities
+
+## Table View Enhancements
+
+The flow and packet capture keeps the same abilities, showing the flow table, cycling between `displays` and `enrichments` and applying `live filters`.
+
+![table](table.png)<br>
+Figure 2: Flow table view
+
+New in 1.10:
+- *Customizable Columns*: Click the `Manage Columns` button to select which columns to display and tailor the table to your needs.<br><br>
+![manage columns](columns.png)<br>
+Figure 3: Column selection popup
+
+- *Smart Filtering*: Live filters now include auto-suggestions, making it easier to select the right keys and values.<br><br>
```

Comment:
It's a small detail, but the bullet is aligned left and the text seems to be aligned to the center. I think would be nicer for the text to be right after the bullet.